Description: zero terminate getopt_long() argument
 Avoid crash on unknown command line arguments by
 making sure the array passed to getopt_long is null
 terminated.
Author: Petter Reinholdtsen <pere@hungry.com>
Bug-Debian: http://bugs.debian.org/716525
Forwarded: no
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2014-10-22

Index: oggvideotools/src/binaries/oggThumb.cpp
===================================================================
--- oggvideotools.orig/src/binaries/oggThumb.cpp	2016-05-24 10:38:11.515726684 +0200
+++ oggvideotools/src/binaries/oggThumb.cpp	2016-05-24 10:38:27.603862021 +0200
@@ -262,7 +262,8 @@
       /* has_arg: */    1,
       /* flag:    */    NULL,
       /* val:     */    opt_verbose
-    }
+    },
+    {0},
   };
 //#endif
 
